### ISSUE TITLE
Price information missing in most requests

### ISSUE THREAD

**Issue by stijnmeijer** [derhuerst/db-rest#59]
First of all, what a great API. You did a great job on this one by packing 80% of the useful data whilst not over-complicating it. 

During recent weeks I noticed that price information is missing more and more often. Currently it seems that almost all price fields are set to €-0,01 even though prices are available in the app. 
Take a look at the example request from the documentation: https://v6.db.transport.rest/journeys?from=8011113&to=8010159&departure=tomorrow+2pm&results=2
At the time of writing, the price->amount field was equal to -0.01 . This applied to many routes. 

I suspect this would be an upstream problem, but haven't had the chance to verify.

**Comment by traines-source** [comment]
See this related issue here: https://github.com/public-transport/hafas-client/issues/331

TL;DR: Prices won't return in the DB HAFAS and it looks like it might be completely shut down soon. A possible drop-in replacement relying on the new DB APIs is https://github.com/public-transport/db-vendo-client

**Comment by derhuerst** [comment]
Closing this in favour of https://github.com/public-transport/hafas-client/issues/331.
